docs/Community/triage/README.md

```diff
@@ -0,0 +1,31 @@
+Triage
+======
+
+https://en.wikipedia.org/wiki/Triage
+
+    Triage is the process of determining the priority of patients' treatments
+    based on the severity of their condition or likelihood of recovery with and
+    without treatment.
```

Comment:
I suggest pointing to https://en.wikipedia.org/wiki/Software_bug#Bug_management instead. Although I understand this may be where the term originates, I'm not sure it will be helpful to someone who hasn't come across the term before.

Reply:
The word "triage" is nowhere to be found on that page, though it is a better description of what we do in the triage meeting.

I think it is useful, especially for folks who have not come across the term "triage" before (eg, non-native english speakers) to understand the context it conveys, and I also agree that providing a link to deeper description of Bug Management is helpful. Thanks.

Reply:
Yes, you're right, I got there through https://en.wikipedia.org/wiki/Triage_(disambiguation) and "Bug Triage", but I see the landing location doesn't mention the word, so it's not that helpful.